src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -62,11 +62,19 @@
 (defn create-form-label [text]
   [:div {:style {:marginBottom "0.16667em" :fontSize "88%"}} text])
 
+;; jQuery3+React warns about nil :value elements. Replace with empty string.
+(defn- replace-nil-value-key [props]
+  (reduce-kv (fn [m k v]
+               (if (and (= k :value) (nil? v))
+                 (assoc m k "")
+                 (assoc m k v)))
+             {} props))
```

Comment:
That actually does not work. It was my first approach (if you look at the commits individually). That ends up causing a different problem. In the case where we remove a nil `:value` property, and some other component then tries to manipulate the value, react complains about changing a component from controlled to uncontrolled and vice versa: https://facebook.github.io/react/docs/forms.html#controlled-components